backend/swift/swift.go

```diff
@@ -120,6 +120,10 @@ func init() {
 				Help:  "Admin",
 				Value: "admin",
 			}},
+		}, {
```

Comment:
It would be nice to have some example storage policies here so users know what to type.

Either some `Examples` or another line in the Help with some text.  Seperate it with `\n` otherwise it will appear in `rclone --help`

Reply:
I'll add some examples.
I could only find OVH implementing its cold storage according to the "swift standard" [0]
It seems that Rackspace does not offer cold storage.
And IBM has implemented this as a separate "location" [1], e.g. eu-vault / eu-cold

[0]https://docs.openstack.org/swift/latest/overview_policies.html
[1]https://console.bluemix.net/docs/services/cloud-object-storage/basics/classes.html#use-storage-classes

OTH, this is the equivalent of the storage classes on S3 and GCS, where these options are non-advanced too. So for the sake of consistency I'd keep this option out of the advanced options.